src/main/java/at/ac/fhcampuswien/fhmdb/HomeController.java

```diff
@@ -50,10 +53,32 @@ public class HomeController {
 
     private final ObservableList<Movie> movies = FXCollections.observableArrayList();
 
+    @FXML
+    private VBox navigation;
+
+    @FXML
+    private JFXButton navigationButton;
+
+    @FXML
+    private JFXButton aboutButton;
+
+    private SVGPath cross = new SVGPath();
+    private SVGPath burger = new SVGPath();
+
     public void initialize() {
         movies.addAll(getAllMoviesOrEmptyList());
         sortMovies();
 
+        burger.setContent("M 10 10 L 30 10 M 10 20 L 30 20 M 10 30 L 30 30");
```

Comment:
I would save these two (lines 72 and 76) strings for SVG icons in some constants

Reply:
Moved to SVG class in UI package.